tests/test_text_signature.rs

```diff
@@ -67,6 +68,7 @@ fn class_with_docs_and_signature() {
 }
 
 #[test]
+#[cfg_attr(Py_LIMITED_API, ignore)]
```

Comment:
Can you remind me again what the issues with `#[text_signature]` are and whether there is a likely way we can fix it?

Reply:
The issue with `text_signature` is that in Python<3.10 `tp_doc` gets special handling that makes it not work.

Reply:
Understood, perhaps this and the other `#[text_signature] tests should be:

```suggestion
#[test]
#[cfg_attr(all(Py_LIMITED_API, not(Py_3_10)), ignore)]
```

Reply:
I believe that's correct, yes.